INSTALL.md

```diff
@@ -56,6 +56,9 @@ Also: GNU make, and Python 2.7 or Python 3.2 or later
 * libvirt-daemon-driver-lxc (if running CTS container tests)
 * libvirt-daemon-lxc (if running CTS container tests)
 * libvirt-login-shell (if running CTS container tests)
+* nmap (if not specifying an IP address base)
+* oprofile (if running CTS profiling tests)
+* dlm (if running CTS tests)
```

Comment:
Interestingly I've never noticed the HAE tests before. Is the dlm dependency specific to those tests, or did you find some other usage? It looks like the HAE tests can use OCFS2 as well.

Reply:
Just in the course of running the tests, it came up as a "command not found" error.  It's tried in StopTest.__call__, after ps.  I couldn't be sure that its absence was causing failures but I wanted to eliminate the error message regardless.

Reply:
OK, that's just to grab some system info for debugging in case something fails (and the node gets rebooted in later tests). We should ignore stderr for the dlm_tool call since it's not actually expected to be there, just something extra.

Maybe "(to log DLM debugging info after CTS tests)"

I believe the presence of DLM also enables the HAE tests, but I don't have any experience with it to be sure.